stripe/src/main/java/com/stripe/android/view/CardInputView.java

```diff
@@ -79,6 +80,28 @@ private void initView() {
         mCardNumberSpace = findViewById(R.id.space_in_container);
         mCardNumberIsViewed = true;
 
+        mErrorColorInt = mCardNumberEditText.getDefaultErrorColorInt();
+        mTintColorInt = mCardNumberEditText.getHintTextColors().getDefaultColor();
+        if (attrs != null) {
+            TypedArray a = getContext().getTheme().obtainStyledAttributes(
+                    attrs,
+                    R.styleable.CardInputView,
+                    0, 0);
+
+            try {
```

Comment:
Note that Google suggests a try-finally block for this, as it can theoretically throw runtime exceptions and we don't want to leak.
https://developer.android.com/training/custom-views/create-view.html#applyattr